cjs/interface/event.js

```diff
@@ -43,11 +43,23 @@ const GlobalEvent = typeof Event === 'function' ?
     }
     
     stopImmediatePropagation() {
-      this.stopPropagation();
       this._stopImmediatePropagationFlag = true;
     }
   };
 
-exports.Event = GlobalEvent;
+
+
+  class DOMEvent extends GlobalEvent {
+    // specs: "set this’s stop propagation flag and this’s stop immediate propagation flag"
+    // https://dom.spec.whatwg.org/#dom-event-stopimmediatepropagation
+    // but Node don't do that so for now we extend it
+    stopImmediatePropagation() {
+      super.stopPropagation();
+      super.stopImmediatePropagation();
+    }
+  }
+  
+
+exports.Event = DOMEvent;
```

Comment:
Yeah ... this is what I meant ... it should be fine, but it might break if `stopImmediatePropagation` is not implemented in `GlobalEvent` ... whatever that is ... maybe a try/catch call to `super.stopImmediatePropagation();` could fix that, but it looks a bit dirty.

Reply:
That's a fair point. I would expect the environment to be Node. Is there any way `Event` could be defined without implementing `stopImmediatePropagation` which is part of the Event spec?

I can add this to be safe:
```javascript
      if (typeof super.stopImmediatePropagation === 'function')
        super.stopImmediatePropagation();
```

Reply:
Added to be on the safe side